language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/json/JsonLfWriter.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.javaapi.data.codegen.json;
+
+import java.io.IOException;
+import java.io.Writer;
+
+public class JsonLfWriter {
```

Comment:
similar to our discussion on decoders, at some point it would be great to also support an object mapper instead of directly going to a string. But fine to not do this in this PR.

Reply:
Yes, I created
https://digitalasset.atlassian.net/browse/LT-39
to track this.